tests/unit/models/requests/test_sign.py

```diff
@@ -72,3 +72,13 @@ def test_valid_seed(self):
             transaction=_TRANSACTION, seed=_SEED, key_type=CryptoAlgorithm.SECP256K1
         )
         self.assertTrue(request.is_valid())
+
+    def test_ctor_has_positional_args(self):
```

Comment:
Why is this test here?

Reply:
Also, what does ctor mean?

Reply:
i shortened constructor to `ctor`. i can use the full name

Reply:
updated in [0cbc07e](https://github.com/XRPLF/xrpl-py/pull/673/commits/0cbc07eca9a5e647bf1be57c4e4403c888edf230).
Let me know if that looks ok